kfs-core/src/main/java/edu/arizona/kfs/fp/document/service/impl/DisbursementVoucherExtractionHelperServiceImpl.java

```diff
@@ -0,0 +1,259 @@
+package edu.arizona.kfs.fp.document.service.impl;
+
+import java.math.BigDecimal;
+import java.sql.Date;
+import java.util.List;
+
+import org.apache.commons.lang.StringUtils;
+import org.kuali.kfs.fp.businessobject.DisbursementVoucherNonEmployeeExpense;
+import org.kuali.kfs.fp.businessobject.DisbursementVoucherNonEmployeeTravel;
+import org.kuali.kfs.fp.businessobject.DisbursementVoucherPayeeDetail;
+import org.kuali.kfs.fp.businessobject.DisbursementVoucherPreConferenceDetail;
+import org.kuali.kfs.fp.businessobject.DisbursementVoucherPreConferenceRegistrant;
+import org.kuali.kfs.fp.document.DisbursementVoucherConstants;
+import org.kuali.kfs.fp.document.DisbursementVoucherDocument;
+import org.kuali.kfs.fp.document.service.DisbursementVoucherTaxService;
+import org.kuali.kfs.pdp.PdpParameterConstants;
+import org.kuali.kfs.pdp.businessobject.PaymentAccountDetail;
+import org.kuali.kfs.pdp.businessobject.PaymentDetail;
+import org.kuali.kfs.pdp.businessobject.PaymentNoteText;
+import org.kuali.kfs.sys.KFSConstants;
+import org.kuali.kfs.sys.businessobject.SourceAccountingLine;
+import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.kfs.sys.service.impl.KfsParameterConstants;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+import org.kuali.rice.core.api.util.type.KualiInteger;
+import org.springframework.transaction.annotation.Transactional;
+
+@Transactional
+public class DisbursementVoucherExtractionHelperServiceImpl extends org.kuali.kfs.fp.document.service.impl.DisbursementVoucherExtractionHelperServiceImpl {

```

Comment:
That must have gotten accidentally reverted when I was cleaning up code. This has been fixed.